tests/README.md

```diff
@@ -1,92 +1,25 @@
-# System Testing
-
-The tests in this directory are meant to be run on a live Kubernetes environment to verify a real system. These
```

Comment:
I would like to still include some context here as to what lives in this directory. Explain the difference between the types of tests that exist here, since it can be confusing.

Reply:
A ToC would also be nice since there is a lot in this README now.

Reply:
I added a little bit...with the help of copilot, not sure who is worse at writing docs between the two of us 😅